portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,22 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
```

Comment:
I know this line was already here (and provided by SPS, right?), so no need to change it, but for future reference, I would have recommended SPS use something more descriptive here than "/data". "/data" is so general, it's like going to a restaurant and ordering "food". What kind of data are we sending? What is it for? Personally, I would have put something like "/greetings" or something.

Reply:
Yeah for sure, the vagueness of it just being called data confused me on the functionality of that section of the module